aws-lc-rs/src/lib.rs

```diff
@@ -148,18 +149,21 @@ pub fn init() {
 }
 
 #[cfg(feature = "fips")]
+#[deprecated = "See `crate::get_fips_service_status`"]
```

Comment:
`get_fips_service_status` is only exposed as `pub(crate)` within the `fips` module. It's also only built when the "fips" feature is enabled.  What's the plan for exposing this function?